File: Presentation/ObjC-Swift 混编在模块化:组件化项目中的实践/sample_projects/using_dependency_as_source_code/MainProject/ViewController.m
//
//  ViewController.m
//  MainProject
//
//  Created by ShannonChen on 2020/3/28.
//  Copyright © 2020 ShannonChen. All rights reserved.
//

#import "ViewController.h"
#import "ObjCLibA.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    self.view.backgroundColor = [UIColor blueColor];
    
    [[ObjCLibA new] sayHello];
}


@end

File: WorldTrotter/WorldTrotter/ViewController.swift
//
//  ViewController.swift
//  WorldTrotter
//
//  Created by ShannonChen on 2017/2/5.
//  Copyright © 2017年 ShannonChen. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let firstFrame = CGRect(x: 160, y: 240, width: 100, height: 150)
        let firstView = UIView.init(frame: firstFrame)
        firstView.backgroundColor = UIColor.blue
        view.addSubview(firstView)
    }



}

